packages/frontend/src/components/UserSettings/PasswordPanel/index.tsx

```diff
@@ -1,12 +1,22 @@
+import { validatePassword } from '@lightdash/common';
+import { PasswordValidationResult } from '@lightdash/common/src/types/passwordValidationResult';
```

Comment:
This import is wrong, you are importing this type from `src` (not from dist). It should be like `validatePassword`